mne/preprocessing/ica.py

```diff
@@ -2624,3 +2638,39 @@ def corrmap(icas, template, threshold="auto", label=None, ch_type="eeg",
         return template_fig, labelled_ics
     else:
         return None
+
+
+def read_ica_eeglab(file_name):
```

Comment:
input are commonly called `fname`

Reply:
file_name changed to fname in read_ica_eeglab and test_read_ica_eeglab.